editor/editor_atlas_packer.cpp

```diff
@@ -105,7 +105,7 @@ void EditorAtlasPacker::chart_pack(Vector<Chart> &charts, int &r_width, int &r_h
 
 		Ref<BitMap> src_bitmap;
 		src_bitmap.instance();
-		src_bitmap->create(aabb.size / divide_by);
+		src_bitmap->create(Size2(MAX(aabb.size.x / divide_by, 1), MAX(aabb.size.y / divide_by, 1)));
```

Comment:
I'm not familiar with the atlas packer, but was wondering whether this should be using `Math::ceil` and rounding up, rather than just dealing with the zero case?

Reply:
That's a good point, will look into it

Reply:
Updated to try this, if someone can check it out